go.mod

```diff
@@ -15,7 +15,7 @@ require (
 	github.com/google/gopacket v1.1.19
 	github.com/hashicorp/golang-lru v0.5.4
 	github.com/hashicorp/golang-lru/v2 v2.0.2
-	github.com/kubernetes/cri-api v0.27.1
+	github.com/kubernetes/cri-api v0.23.5-rc.0
```

Comment:
I think I used the replace because I couldn't get the imports working. I think we _should_ use the newer versions and remove the replace instead if possible.

Reply:
Don't merge yet. Making changes and will trigger a E2E test. Then squash the commits.